### DISCUSSION TITLE
Allow all dropdown menus to remain open without having to hold down the mouse

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#22489] [Feature requests] [answered]
This was a welcome change with the cheats dropdown and perhaps some others, but I noticed that not all menus remain open without having to hold down the mouse button. Could this be enabled for all dropdown menus? Specifically the sources dropdown in the object selection would be helpful as well as the visibility dropdown and show map button/plugins.

**Comment by J-PIE-314** [comment, marked as answer]
This feature is available right now, but only if enlarged UI is enabled.

![photo of options](https://github.com/user-attachments/assets/f8cc80d0-022a-4b60-bd2f-68284661c144)
